src/js/views/createCertificate/translations/pt_br.createCertificate.i18n.json

```diff
@@ -16,16 +16,16 @@
       "step1": "Passo 1: ",
       "step1Text": "Execute o comando: openssl req  -newkey 'rsa:2048'  -nodes  -sha256  -keyout 'privatekey.pem'  -out 'csr.pem'  -subj '/CN=Any value'",
       "step2": "Passo 2: ",
-      "step2Text": "Copie e conteúdo do arquivo csr.pem para a area de texto abaixo."
+      "step2Text": "Copie e cole o conteúdo do arquivo csr.pem para a area de texto abaixo."
     },
     "generateCertificate": "Gerar Certificado",
-    "inputPlaceholder": "Insira aqui seu texto"
+    "inputPlaceholder": "Copie e cole o conteúdo do arquivo .pem aqui."
```

Comment:
I think that this text should be `Cole o conteúdo do arquivo .pem aqui.`, because the label already tells the user to copy.